### ISSUE TITLE
Add option to show only text (no symbols)

### ISSUE THREAD

**Issue by mattrobineau** [onsails/lspkind-nvim#34]
I use neovim on multiple OSes and multiple PCs and I don't feel inclined to download and install a font on every machine. This means that all the symbols are usually broken. 

I would suggest adding a `text_only` option rather than having a configuration that sets all the `symbol_map` fields to an empty string ""

Configuration would look like this:

```
-- nvim-cmp setting:
    formatting = {
        format = lspkind.cmp_format({
            text_only = true,
            menu = ({
                buffer = "[buff]",
                nvim_lsp = "[lsp]",
                luasnip = "[snip]",
                path = "[path]"
            })
        })
    }

-- default lsp config

require('lspkind').init({
    text_only = false,
    -- [... snip other config stuff ...]
})
```

Note that when `text_only` is set it would override the `with_text` setting since we can't have `with_text = false` when showing only text.

The popup would look like this:

![popup example](https://user-images.githubusercontent.com/427379/151679849-c4d3101d-d562-4587-b02e-e3b1a6b95708.png)

You can find a working example in [this fork](https://github.com/mattrobineau/lspkind-nvim).

**Comment by onsails** [comment]
What if we introduce a `mode` option with possible values `text`, `text_symbol`, `symbol_text`, `symbol`? related to #28

**Comment by mattrobineau** [comment]
@onsails I think its a good idea. Do we remove `with_text` and add the `mode` option or leave it for backwards compatibility?

**Comment by onsails** [comment]
I'd leave this option with a deprecation error notice.

**Comment by onsails** [comment]
fixed in #35
